examples/client/client/common.nim

```diff
@@ -5,12 +5,12 @@ import # vendor libs
   task_runner
 
 import # status lib
-  ../../../status/private/accounts/public_accounts
+  ../../../status/api/accounts
```

Comment:
Shouldn't this now be available via `import status/api/accounts`?

Reply:
We have to use a relative path for `status` in modules under `examples/` for the same reason we have to do so for modules under `test/`.

The nimbus-build-system (its `env.sh` plus work it does when invoking `make` targets) allows for the Nim compiler / nimble to load the modules under `vendor/` as if they were installed in a conventional nimble setup. But apart from dependencies and transitive nimbus-build-system dependencies under `vendor/`, it does nothing for us with regard to `examples/` and `test/` in relation to `status.im` / `status/` in the root of the repo.

Reply:
That being said, if someone using the nimbus-build-system installed nim-status in their `vendor/` (or it was one of their transitive deps under `vendor/`), then they would be able to `import status` or `import status/api/accounts`, etc.

Reply:
Makes complete senese. I was thinking as this is an example of client consuming `nim-status`, that it serves to show potentials consumers of `nim-status` (as an external dependency using the nimbus-build-system) how they could simply `import status/api/accounts`. Maybe it might be worth a comment in the source so that devs looking to implement their own client will know they can do it that way if they weren't aware. Of course this is something we can put in the README as well.

Reply:
I filed an enhancement issue for nimbus-build-system: https://github.com/status-im/nimbus-build-system/issues/24.

Reply:
See PR #262.

🚀 Thanks @stefantalpalaru!